### ISSUE TITLE
[FEATURE] Personal settings - calendar/date localization

### ISSUE THREAD

**Issue by madarasz** [madarasz/always-be-running#4]
**User story**: As a ABR user, I want a personal setting for the first day of the week and date format.

**More details**: These should be settings in the personal page. This affects calendars and displayed dates.
**Development effort**: medium

**Things to consider**
- The ISO date format (YYYY.MM.DD) is nice. This is used in the system itself, since it's easy to sort.
- The ISO standard for first day of week is Monday. This is the default in Europe, most of Asia and Australia. Sunday is the first day in USA, Canada, most other American countries, Japan and some South African countries. Saturday is the first day in most Middle Eastern countries and some North African countries.
- Trying to guess the user's location by IP and then deciding on these settings are overly complicated.

**Comment by Smashman** [comment]
You can get some of these from the browser itself.
Might be worth considering.

**Comment by 5N00P1** [comment]
or the user can just define it?
Travelling to the US, I don't want to have this changed!

**Comment by Smashman** [comment]
The browser​ is set by the user.

**Comment by 5N00P1** [comment]
I agree, but he was saying he would try to guess the users location by IP...
